cra-rxjs-styled-components/src/components/issue/Issue/Issue.view.tsx

```diff
@@ -0,0 +1,35 @@
+import IssueTabHeader from '../IssueTabHeader';
+import IssueCard from '../IssueCard';
+import { PaginationContainer, Content, Wrapper } from './Issue.style';
+import type { Issue } from './Issue.type';
+import type { IssueTabValues } from '../types';
+
+type IssueProps = {
+  issues: Issue[];
+  changeActiveTab: (value: IssueTabValues) => void;
+};
+
+export default function IssueView({ issues, changeActiveTab }: IssueProps) {
+  return (
+    <Wrapper>
+      <Content>
+        <IssueTabHeader toggleTab={changeActiveTab} />
+        {issues.map((issue, index) => (
+          <IssueCard
+            title={issue.title}
+            openedNum={issue.openedNum}
+            openedDay={issue.openedDay}
+            openedBy={issue.openedBy}
+            status={issue.status}
+            messageCount={issue.messageCount}
+            key={index}
+          />
+        ))}
+      </Content>
+      <PaginationContainer>
+        <span className="prev">Previous</span>
+        <span className="next">Next</span>
+      </PaginationContainer>
```

Comment:
Are the controls for the pagination living within the container component?

Reply:
Yes they are. At the moment they do nothing. What were your expectations?

Reply:
I assumed they'd paginate, but if that's out of scope for this ticket you can disregard it.

Reply:
That's the scope for this ticket. :)